icons/v8/v8-original.svg

```diff
@@ -0,0 +1,100 @@
+</g>
+</g>
+<g clip-path="url(#clip13)" clip-rule="nonzero">
+<g clip-path="url(#clip14)" clip-rule="nonzero">
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:0.2;" d="M 87.273438 34.15625 L 87.273438 34.5 C 87.082031 21.800781 76.730469 11.566406 64 11.566406 C 51.269531 11.566406 40.910156 21.800781 40.726562 34.5 L 40.726562 34.15625 C 40.726562 21.300781 51.144531 10.882812 64 10.882812 C 76.855469 10.882812 87.273438 21.300781 87.273438 34.15625 Z M 44.765625 47.949219 C 38.125 53.457031 33.882812 61.796875 33.882812 71.117188 L 33.882812 71.460938 C 33.984375 62.136719 38.332031 53.835938 45.066406 48.378906 L 45.382812 48.125 L 45.066406 47.714844 L 44.765625 47.988281 Z M 83.234375 47.949219 L 82.933594 47.695312 L 82.617188 48.125 L 82.933594 48.402344 C 89.675781 53.875 94.023438 62.160156 94.117188 71.46875 L 94.117188 71.125 C 94.117188 61.816406 89.875 53.464844 83.234375 47.988281 Z M 83.234375 47.949219 "/>
+</g>
+</g>
+<g clip-path="url(#clip15)" clip-rule="nonzero">
+<g clip-path="url(#clip16)" clip-rule="nonzero">
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(14.901961%,19.607843%,21.960784%);fill-opacity:0.2;" d="M 94.117188 70.777344 L 94.117188 71.117188 C 94.117188 87.75 80.632812 101.238281 64 101.238281 C 47.367188 101.238281 33.882812 87.75 33.882812 71.117188 L 33.882812 70.777344 C 34.066406 87.253906 47.484375 100.550781 64 100.550781 C 80.515625 100.550781 93.933594 87.253906 94.117188 70.777344 Z M 40.726562 33.8125 L 40.726562 34.15625 C 40.71875 39.191406 42.351562 44.097656 45.382812 48.121094 C 45.492188 48.050781 45.601562 47.957031 45.722656 47.875 C 42.554688 43.867188 40.796875 38.921875 40.726562 33.8125 Z M 87.273438 33.8125 C 87.203125 38.921875 85.445312 43.867188 82.277344 47.875 C 82.398438 47.957031 82.507812 48.042969 82.617188 48.125 C 85.539062 44.292969 87.273438 39.433594 87.273438 34.164062 Z M 87.273438 33.8125 "/>
+</g>
+</g>
+<path style=" stroke:none;fill-rule:nonzero;fill:url(#radial0);" d="M 124.234375 21.835938 L 113.285156 41 L 110.203125 46.476562 L 102.742188 46.476562 L 93.363281 64.34375 C 96.777344 79.289062 88.449219 94.417969 73.992188 99.523438 L 64 117.664062 L 54.074219 99.523438 L 53.046875 99.183594 C 39.121094 93.703125 31.292969 78.863281 34.636719 64.273438 L 25.257812 46.476562 L 17.867188 46.476562 L 14.714844 41 L 3.765625 21.835938 L 35.25 21.835938 L 40.386719 32.035156 L 40.660156 32.171875 L 40.726562 32.035156 C 41.660156 19.910156 51.769531 10.546875 63.929688 10.546875 C 76.09375 10.546875 86.203125 19.910156 87.136719 32.035156 L 87 32.171875 L 86.929688 32.035156 L 92.75 21.835938 Z M 124.234375 21.835938 "/>
+</g>
+</svg>
```

Comment:
How does this look?

![image](https://user-images.githubusercontent.com/51391473/186228326-0dd4ff7c-6b97-420c-acbd-d0a4ff37c9a8.png)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><defs><clipPath id="a"><path d="M14 40h100v7H14Zm0 0"/></clipPath><clipPath id="b"><path d="M124.234 21.836 113.285 41l-3.082 5.477h-7.39l-9.45 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></clipPath><clipPath id="c"><path d="M3 21h122v97H3Zm0 0"/></clipPath><clipPath id="d"><path d="M124.234 21.836 113.285 41l-3.082 5.477h-7.39l-9.45 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></clipPath><clipPath id="e"><path d="M3 21h122v2H3Zm0 0"/></clipPath><clipPath id="f"><path d="M124.234 21.836 113.285 41l-3.082 5.477h-7.39l-9.45 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></clipPath><clipPath id="g"><path d="M17 45h94v73H17Zm0 0"/></clipPath><clipPath id="h"><path d="M124.234 21.836 113.285 41l-3.082 5.477h-7.39l-9.45 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></clipPath><clipPath id="i"><path d="M52 25h24v58H52Zm0 0"/></clipPath><clipPath id="j"><path d="M124.234 21.836 113.285 41l-3.082 5.477h-7.39l-9.45 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></clipPath><clipPath id="k"><path d="M33 10h62v92H33Zm0 0"/></clipPath><clipPath id="l"><path d="M124.234 21.836 113.285 41l-3.082 5.477h-7.39l-9.45 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></clipPath><clipPath id="m"><path d="M33 10h62v62H33Zm0 0"/></clipPath><clipPath id="n"><path d="M124.234 21.836 113.285 41l-3.082 5.477h-7.39l-9.45 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></clipPath><clipPath id="o"><path d="M33 33h62v69H33Zm0 0"/></clipPath><clipPath id="p"><path d="M124.234 21.836 113.285 41l-3.082 5.477h-7.39l-9.45 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></clipPath><radialGradient id="q" gradientUnits="userSpaceOnUse" cx="78" cy="337" fx="78" fy="337" r="1854" gradientTransform="rotate(.277) scale(.06845)"><stop offset="0" style="stop-color:#fff;stop-opacity:.101961"/><stop offset="1" style="stop-color:#fff;stop-opacity:0"/></radialGradient></defs><g clip-path="url(#a)"><g clip-path="url(#b)"><path style="stroke:none;fill-rule:nonzero;fill:#9e9e9e;fill-opacity:1" d="m104.727 40.316-2.942 6.16h8.348l.414-.683 3.078-5.477h-8.965Zm-81.59 0h-8.762l3.492 6.16h8.281Zm0 0"/></g></g><g clip-path="url(#c)"><g clip-path="url(#d)"><path style="stroke:none;fill-rule:nonzero;fill:#424242;fill-opacity:1" d="M124.234 21.836 113.285 41h-7.668L93.363 64.344 64 117.664l-29.363-53.39L22.383 41h-7.668L3.765 21.836H35.25l5.137 10.2 16.699 30.456L63.93 75.02l6.574-12.528L86.93 32.035l5.82-10.2"/></g></g><g clip-path="url(#e)"><g clip-path="url(#f)"><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:.2" d="M35.633 22.52H4.148l-.382-.684H35.25Zm57.117-.684-.379.684h31.473l.39-.684Zm0 0"/></g></g><g clip-path="url(#g)"><g clip-path="url(#h)"><path style="stroke:none;fill-rule:nonzero;fill:#263238;fill-opacity:.2" d="m110.574 45.777-.398.7h-7.434l-9.398 17.832L64 117.664 34.656 64.293l-9.379-17.816h-7.441l-.383-.684h7.824l.364.684 9.015 17.132L64 116.98l29.344-53.355 9.406-17.848"/></g></g><g clip-path="url(#i)"><g clip-path="url(#j)"><path style="fill-opacity:0" d="M75.637 71.117c0 6.43-5.211 11.637-11.637 11.637s-11.637-5.207-11.637-11.637c0-6.426 5.211-11.637 11.637-11.637s11.637 5.211 11.637 11.637Zm-1.371-35.594c0 5.672-4.594 10.27-10.266 10.27s-10.266-4.598-10.266-10.27c0-5.668 4.594-10.265 10.266-10.265s10.266 4.597 10.266 10.265Zm0 0" fill="none"/></g></g><g clip-path="url(#k)"><g clip-path="url(#l)"><path style="stroke:none;fill-rule:nonzero;fill:#4285f4;fill-opacity:1" d="M93.363 64.344A30.215 30.215 0 0 0 82.961 47.71a23.274 23.274 0 0 0-24.805-36.07 23.277 23.277 0 0 0-17.293 20.394 23.225 23.225 0 0 0 4.176 15.676 30.027 30.027 0 0 0-10.402 16.562 29.69 29.69 0 0 0-.754 6.844c0 12.735 7.941 23.684 19.164 28.067A30.195 30.195 0 0 0 64 101.238a30.045 30.045 0 0 0 23.617-11.34 30.047 30.047 0 0 0 5.746-25.554ZM64 25.94c5.293 0 9.582 4.293 9.582 9.582 0 5.293-4.289 9.586-9.582 9.586-5.293 0-9.582-4.293-9.582-9.586 0-5.289 4.289-9.582 9.582-9.582Zm0 56.13c-6.023 0-10.953-4.93-10.953-10.954a10.956 10.956 0 0 1 6.148-9.808 10.957 10.957 0 0 1 11.512 1.183 10.895 10.895 0 0 1 4.246 8.625c0 6.051-4.906 10.953-10.953 10.953Zm0 0"/></g></g><g clip-path="url(#m)"><g clip-path="url(#n)"><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:.2" d="M87.273 34.156v.344C87.082 21.8 76.73 11.566 64 11.566S40.91 21.801 40.727 34.5v-.344c0-12.855 10.418-23.273 23.273-23.273S87.273 21.3 87.273 34.156ZM44.766 47.95c-6.641 5.508-10.883 13.848-10.883 23.168v.344c.101-9.324 4.449-17.625 11.183-23.082l.317-.254-.317-.41-.3.273Zm38.468 0-.3-.254-.317.43.317.277c6.742 5.473 11.09 13.758 11.183 23.067v-.344c0-9.309-4.242-17.66-10.883-23.137Zm0 0"/></g></g><g clip-path="url(#o)"><g clip-path="url(#p)"><path style="stroke:none;fill-rule:nonzero;fill:#263238;fill-opacity:.2" d="M94.117 70.777v.34c0 16.633-13.484 30.121-30.117 30.121-16.633 0-30.117-13.488-30.117-30.12v-.34C34.066 87.253 47.484 100.55 64 100.55s29.934-13.297 30.117-29.774Zm-53.39-36.965v.344a23.152 23.152 0 0 0 4.656 13.965c.11-.07.219-.164.34-.246a23.197 23.197 0 0 1-4.996-14.063Zm46.546 0a23.197 23.197 0 0 1-4.996 14.063c.121.082.23.168.34.25 2.922-3.832 4.656-8.691 4.656-13.96Zm0 0"/></g></g><path style="stroke:none;fill-rule:nonzero;fill:url(#q)" d="M124.234 21.836 113.285 41l-3.082 5.477h-7.46l-9.38 17.867c3.414 14.945-4.914 30.074-19.37 35.18L64 117.663l-9.926-18.14-1.027-.34c-13.926-5.48-21.754-20.32-18.41-34.91l-9.38-17.797h-7.39L14.715 41 3.765 21.836H35.25l5.137 10.2.273.136.067-.137C41.66 19.91 51.77 10.547 63.93 10.547c12.164 0 22.273 9.363 23.207 21.488l-.137.137-.07-.137 5.82-10.2Zm0 0"/></svg>
```

Reply:
> How does this look?
> 
> ![image](https://user-images.githubusercontent.com/51391473/186228326-0dd4ff7c-6b97-420c-acbd-d0a4ff37c9a8.png)

i like